helm/hpcc/values.schema.json

```diff
@@ -740,6 +749,12 @@
         "logging": {
           "$ref": "#/definitions/logging"
         },
+        "auth": {
+          "$ref": "#/definitions/auth"
+        },
```

Comment:
Unless you are going to do your change to move it to the ldap section, add ldapAddress here, otherwise there is no way of setting it.

Reply:
ldapAddress is being moved into ldap config with https://track.hpccsystems.com/browse/HPCC-26455